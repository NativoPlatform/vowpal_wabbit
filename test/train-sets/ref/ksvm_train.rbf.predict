0
0.149683
0.007743
-0.139958
-0.242241
-0.282828
-0.224890
-0.274963
-0.343955
-0.396932
-0.351765
-0.375836
-0.430253
-0.468264
-0.471225
-0.397705
-0.406749
-0.342908
-0.427083
-0.415860
-0.471277
-0.415454
-0.372059
-0.336644
-0.284620
-0.372080
-0.439196
-0.538095
-0.433416
-0.396221
-0.478485
-0.371771
-0.418940
-0.468328
-0.470085
-0.458646
-0.459060
-0.445606
-0.415588
-0.299152
-0.318956
-0.344774
-0.367805
-0.355692
-0.452483
-0.482642
-0.407155
-0.417041
-0.497249
-0.418914
-0.458868
-0.417494
-0.416519
-0.446481
-0.421256
-0.291992
-0.456120
-0.475617
-0.462638
-0.419844
-0.408216
-0.389389
-0.237217
-0.544814
-0.404081
-0.433344
-0.279462
-0.392094
-0.541814
-0.443839
-0.329288
-0.252624
-0.403388
-0.426277
-0.233803
-0.296192
-0.319596
-0.462940
-0.409655
-0.444956
-0.449201
-0.452098
-0.304272
-0.413763
-0.520965
-0.501837
-0.486704
-0.376895
-0.355514
-0.609571
-0.429304
-0.401254
-0.143135
-0.334957
-0.245553
-0.489838
-0.405195
-0.277199
-0.449365
-0.394544
-0.287703
-0.324306
-0.313217
-0.330298
-0.277070
-0.302469
-0.375690
-0.442938
-0.413442
-0.336684
-0.262134
-0.274052
-0.337047
-0.398341
-0.195720
-0.396961
-0.399366
-0.351081
-0.383897
-0.423943
-0.339904
-0.311027
-0.289657
-0.414036
-0.225663
-0.134456
-0.286305
-0.179765
-0.445551
-0.329821
-0.267922
-0.262634
-0.245523
-0.325733
-0.335008
-0.418649
-0.280039
-0.212377
0.099761
-0.276152
-0.384391
-0.462294
0.034360
0.328395
-0.257544
-0.076680
-0.221577
0.173069
-0.438621
-0.300683
-0.060943
-0.205070
0.013524
-0.232947
-0.408455
-0.122640
0.230754
-0.193110
0.218176
-0.193559
0.465816
-0.362133
-0.337732
-0.043610
-0.216442
-0.197359
0.224680
-0.295912
-0.283631
0.221608
0.139241
-0.313661
-0.325369
-0.039054
-0.320199
-0.161719
-0.492662
0.193872
-0.069400
0.241125
-0.094545
-0.018284
-0.015343
-0.315921
-0.231248
-0.133065
-0.046507
-0.297086
-0.271563
0.002782
0.252362
-0.103220
-0.237035
-0.249841
-0.233203
-0.177799
-0.066494
-0.278578
-0.161376
-0.153890
-0.231004
-0.087006
-0.069937
-0.276381
-0.324276
-0.363127
0.038065
-0.126797
-0.272179
-0.123166
-0.141258
-0.422560
-0.356973
-0.158917
0.813271
-0.440652
-0.199437
-0.310653
-0.231771
-0.291740
-0.198582
-0.479320
-0.343444
-0.318686
-0.106649
-0.275540
0.820386
0.117768
-0.292023
-0.403816
-0.505864
-0.514217
-0.485692
-0.207857
-0.016518
-0.444669
-0.408933
0.033534
0.551971
-0.078657
-0.007721
-0.236692
-0.465086
0.283144
-0.183479
-0.046673
0.026792
-0.271445
-0.476949
0.141236
