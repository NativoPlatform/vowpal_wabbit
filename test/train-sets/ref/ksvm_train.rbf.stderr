using l2 regularization = 1
predictions = ksvm_train.rbf.predict
Lambda = 1
Kernel = rbf
bandwidth = 1
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.074842 1.149683            2            2.0  -1.0000   0.1497      103
1.004367 0.933892            4            4.0  -1.0000  -0.1400      134
0.944775 0.885184            8            8.0  -1.0000  -0.2750      145
0.928374 0.911974           16           16.0   1.0000  -0.3977       23
0.919879 0.911383           32           32.0  -1.0000  -0.3718       31
0.904293 0.888707           64           64.0  -1.0000  -0.5448       60
0.926975 0.949657          128          128.0   1.0000  -0.1798      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.892674
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 250
Number of kernel evaluations = 195491 Number of cache queries = 170253
Total loss = 223.168411
Done freeing model
Done freeing kernel params
Done with finish 
